assignments/prototypes.js

```diff
@@ -20,6 +20,7 @@ function GameObject(attributes) {
   this.name = attributes.name;
   this.dimensions = attributes.dimensions;
 }
+
 GameObject.prototype.destroy = function () {
   console.log(`${this.name} was removed from the game.`)
```

Comment:
Remember you want this displayed on the players screen. They don't see the console. Only developers do. 

Return would have been a better choice.

Reply:
got it thank you!